operator/option/config.go

```diff
@@ -76,6 +76,9 @@ const (
 	// PProf enabled pprof debugging endpoint
 	PProf = "pprof"
 
+	// PProfPort is the port that the pprof listens on
+	PProfPort = "operator-pprof-port"
```

Comment:
This doesn't need to be called out specifically for operator. This is leftover from when I was considering Helm changes. I'll change this back to `pprof-port`.